pdfmaker/font_reader.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from fontTools import ttLib
```

Comment:
It's for grabbing the font name from a ttf. I can see if there's a way to do this without TTFont.